packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -19,6 +19,12 @@ const PACKAGE_NAME = '@astrojs/vercel/serverless';
 export const ASTRO_LOCALS_HEADER = 'x-astro-locals';
 export const VERCEL_EDGE_MIDDLEWARE_FILE = 'vercel-edge-middleware';
 
+const SUPPORTED_NODE_VERSIONS: Record<string, { status: 'current' } | { status: 'deprecated', removal: string }> = {
+	14: { status: 'deprecated', removal: 'August 15th 2023'  },
+	16: { status: 'deprecated', removal: 'February 6th 2024' },
```

Comment:
Ah the US date format 😅

Reply:
Using Intl.DateTimeFormat now 🫡

Reply:
Meant that mostly as a joke, but now it's even better! Thanks 🙌